packages/labs/compiler/src/test/golden_compiler_test.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * Test with `npm run test -w @lit-labs/compiler`.
+ *
+ * Update goldens with: `npm run update-goldens -w @lit-labs/compiler`.
+ */
+
+import ts from 'typescript';
+import {compileLitTemplates} from '../lib/template-transform.js';
+import {test} from 'uvu';
+// eslint-disable-next-line import/extensions
+import * as assert from 'uvu/assert';
+import {existsSync, readdirSync} from 'fs';
+import {readFile, writeFile} from 'fs/promises';
+import * as url from 'url';
+import * as path from 'path';
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+const TEST_FILES_DIR = path.resolve(__dirname, '../test_files');
+const UPDATE_GOLDENS = process.env['UPDATE_LIT_COMPILER_GOLDENS'] === 'true';
+if (UPDATE_GOLDENS) {
+  const resetColor = '\x1b[0m';
+  const green = '\x1b[32m';
+  const underscore = '\x1b[4m';
+  const bold = '\x1b[1m';
+  console.log(
+    `${green}${underscore}${bold}Updating @lit/compiler goldens${resetColor}`
```

Comment:
Not important, but should it say `@lit-labs/compiler` ?
```suggestion
    `${green}${underscore}${bold}Updating @lit-labs/compiler goldens${resetColor}`
```

Reply:
Great catch! Thank you!